app.env.example

```diff
@@ -92,3 +92,6 @@ CORS_ALLOW_ORIGINS=localhost:8000
 
 # Use IP address or domain_name to replace localhost
 GEO_JSON_URL=http://localhost:3000/provinces.json
+
+# Scorecard max download record
+MAXIMUM_DOWNLOAD_CSV_RECORDS=1000
```

Comment:
@hengsokly please rename MAXIMUM_DOWNLOAD_CSV_RECORDS to MAX_DOWNLOAD_SCORECARD_RECORD